hooks/opositeSideGenerator.js

```diff
@@ -0,0 +1,19 @@
+module.exports = register => {
+    register('generate:before', generator => {
+        if (generator.templateParams && generator.templateParams['generateOppositeSide'] === 'true') {
```

Comment:
```suggestion
        if (generator.templateParams && generator.templateParams['generateOppositeSide'] ) {
```

`generator.templateParams['generateOppositeSide']` is evaluated to true or false

Reply:
No, this will not work. Any parameter from console is `String` and so `-p generateOppositeSide=false` will pass the check since `generator.templateParams['generateOppositeSide']` is defined